docker/tests/test-fedora-binaries.py

```diff
@@ -0,0 +1,50 @@
+# SPDX-FileCopyrightText: 2023 Oxhead Alpha
+# SPDX-License-Identifier: LicenseRef-MIT-OA
+
+import subprocess
+import sys
+
+binaries = [
+    "tezos-client",
+    "tezos-admin-client",
+    "tezos-node",
+    "tezos-signer",
+    "tezos-codec",
+    "tezos-baker-PtMumbai",
+    "tezos-accuser-PtMumbai",
+    "tezos-smart-rollup-client-PtMumbai",
+    "tezos-smart-rollup-node-PtMumbai",
+    "tezos-baker-PtNairob",
+    "tezos-accuser-PtNairob",
+    "tezos-smart-rollup-client-PtNairob",
+    "tezos-smart-rollup-node-PtNairob",
+]
+
+
+def test(cleanup: bool):
```

Comment:
At this point we have to update package index again (on both ubuntu and fedora), because everything fetched by docker `RUN` commands will be cached and reused, so that we won't work on fresh index

Reply:
WDYM?